dataflows/helsinki-transit/README.md

```diff
@@ -1,59 +1,119 @@
 
-# Start MQTT connector using cloud
+# Helsinki Transit Dataflow
 
-## Download Jolt SmartModule
+The Helsinki transit dataflow uses MQTT to ingests data from the Helsinki transit API and then processes the data to provide insights into the transit system. 
 
+The following diagram is a visual representation generated by `sdf`:
+
+<p align="center">
+ <img src="img/helsinki-transit.jpg">
+</p>
+
+The dataflow has two services:
+* `clean-events` removes events with incomplete data
+* `generate-vehicle-stats` generates average speed per vehicle every 5 seconds.
+
+The purpose of the dataflow is to demonstrate how to use the SQL interface to analyze the data.
```

Comment:
I would maybe move this sentence to the top of the document to serve as an introduction.